recovery/main.cpp

```diff
@@ -107,11 +128,9 @@ QString findRecoveryDrive()
         if (!drive.isEmpty())
             break;
     }
-
-    return drive;
+    return(drive);
```

Comment:
Why this change? :wink:

Reply:
Yes, I spotted that too. (Cut and paste from rewritten test program).
I wondered if you would comment..... 👀 😉